### DISCUSSION TITLE
branch or event not found

### DISCUSSION THREAD

**Discussion by hlf20010508** [badges/shields#8739] [Q&A] [answered]
I've worked on it for hours, I need your help.

Here is my repo:
<img width="857" alt="image" src="https://user-images.githubusercontent.com/76218469/208345761-8fd35316-a7a8-4802-90a7-dc4e49261cd4.png">

This is a repo that works fine:
<img width="877" alt="image" src="https://user-images.githubusercontent.com/76218469/208345855-b945b66f-7eb1-496a-bbe6-4946a4100083.png">

I don't know what's the difference.

This is my [yml](https://github.com/hlf20010508/miniopy-async/blob/master/.github/workflows/python-publish.yml)

This is that [yml](https://github.com/xmake-io/xmake/blob/master/.github/workflows/linux.yml)

We both put the yml in `.github/workflows` and branch is `master`, but why mine cannot work?

If there is something I should modify in my yml, please teach me.

Thanks a lot.

**Comment by chris48s** [comment, marked as answer]
This is an example of https://github.com/badges/shields/issues/8736
https://github.com/hlf20010508/miniopy-async/blob/master/.github/workflows/python-publish.yml does not run against a specific branch. You should now be able to solve this by omitting the branch param
